configuration/src/configuration.ts

```diff
@@ -113,6 +122,15 @@ export class Configuration implements IConfiguration {
         if (params.defaultValues !== null) {
             this.providers.push(defaultConfig);
         }
+
+        // Optionally add the keyvault provider:

```

Comment:
We could, but I wouldn't recommend it. KeyVault is suited specifically for secrets, and it's pretty expensive to fetch values from there. Plus, we wouldn't want to clutter up its access logs with non-secrets.